helm/prometheus-rules/templates/alerting-rules/deployment.management-cluster.rules.yml

```diff
@@ -68,7 +68,7 @@ spec:
       annotations:
         description: '{{`Deployment {{ $labels.namespace}}/{{ $labels.deployment }} is not satisfied.`}}'
         opsrecipe: deployment-not-satisfied/
-      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment!~"aws-operator-.+|azure-operator-.+|cluster-operator-.+|coredns-.+|nginx-ingress-controller-.+|worker-.+|master-.+|app-operator-.+|chart-operator-.+|.+-admission-controller-unique|alertmanager.*|prometheus.*|promxy.*", cluster_id!~"argali|giraffe"} > 0
+      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment!~"aws-operator-.+|azure-operator-.+|cluster-operator-.+|coredns-.+|nginx-ingress-controller-.+|worker-.+|master-.+|app-operator-.+|chart-operator-.+|.+-admission-controller|alertmanager.*|prometheus.*|promxy.*", cluster_id!~"argali|giraffe"} > 0
```

Comment:
I've checked on `gauss` that there aren't any deployments using the `-unique` suffix, including admission controllers specifically. Just to be on the safe side, pinging @giantswarm/team-biscuit for your thumbs up